ussologin/ussologin.go

```diff
@@ -69,13 +73,16 @@ var loginForm = form.Form{
 		otpKey: environschema.Attr{
 			Description: "Two-factor auth",
 			Type:        environschema.Tstring,
+			Mandatory:   true,
```

Comment:
Ironically making this mandatory means that it will always contain a string, which can be empty. This is exactly how we want it.